.travis.yml

```diff
@@ -50,6 +47,8 @@ script:
   # start pulling Docker base image in the background
   - nohup docker pull localstack/java-maven-node-python > /dev/null &
   - LAMBDA_EXECUTOR=docker-reuse TEST_PATH="tests/integration/test_lambda.py tests/integration/test_integration.py" make test
+  # build base image
+  - make docker-build-base
```

Comment:
`make docker-build` fails without this in my fork.

Reply:
I can revert, but I don't see any new image on Docker Hub so that the build will fail.

Reply:
The build failed. Could you please push the new image @whummer?